### PR TITLE
Main feature branch of VC creation flow

### PR DESCRIPTION
**Pull request by EmiliaPavlova**
This is the main branch of the 'Create VC Flow' v2 Feature - https://github.com/alkem-io/alkemio/issues/1348

User flow: https://alkem.io/building-alkemio-org/challenges/improvingtheusere-6400/collaboration/vccreateflowwith3typ

UI Designs: https://www.figma.com/design/gV8GGWh99QRKfM65lwedrE/Dashboard-Update?node-id=482-207&t=jCXmCypVZY8hWnEi-1

Fix for https://github.com/alkem-io/client-web/issues/6684 included.

### CODE REVIEW

File: src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql [hunk 1]
```diff
@@ -25,6 +29,20 @@ query NewVirtualContributorMySpaces {
           id
         }
       }
+      community {
```
Comment (ccanos) [inline src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql:32]: community is repeated here

File: src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql [hunk 2]
```diff
@@ -25,6 +29,20 @@ query NewVirtualContributorMySpaces {
...
+        id
+      }
+    }
+  }
+}
+
+query NewSpace($spaceId: UUID_NAMEID!) {
```
Comment (ccanos) [inline src/main/topLevelPages/myDashboard/newVirtualContributorWizard/newVirtualContributorQueries.graphql:39]: I'd call this `SpaceUrl`, and maybe there's already a similar query. 
Also if we have the ID (which I'm pretty sure we do), better access it through the lookup with UUID. if not please call the parameter $spaceNameId

File: src/main/topLevelPages/myDashboard/newVirtualContributorWizard/LoadingState.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  const onCancel = () => {
+    setDialogOpen(true);
+  };
+
+  return (
+    <>
+      <DialogHeader onClose={onCancel} />
+      <Loading text="" />
+      <Gutters padding={gutters(2)} sx={{ textAlign: 'center' }}>
```
Comment (ccanos) [inline src/main/topLevelPages/myDashboard/newVirtualContributorWizard/LoadingState.tsx:26]: I think `textAlign` is a property of `BoxProps` and can be used directly outside `sx`

**Review by ccanos — CHANGES_REQUESTED** [review]
please take a look at these comments

**Review by bobbykolev — APPROVED** [review]

**Review by bobbykolev — APPROVED** [review]

**Review by bobbykolev — APPROVED** [review]

**Review by ccanos — APPROVED** [review]

**Pull request merged**